### PR TITLE
Implement new subscription, ci/cd and terraform 

### PR DESCRIPTION
**Pull request by djorgensendk**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - main
+      - djorgensen/new-sub
```
Comment (djorgensendk) [inline .github/workflows/ci.yml:20]: remove trigger

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+jobs:
+  ci_base:
+    uses: Energinet-DataHub/.github/.github/workflows/ci-base.yml@1.13.0
+  
+  # dotnet_solution_ci:
+  #   uses: Energinet-DataHub/geh-charges/.github/workflows/dotnet-solution-ci.yml@djorgensen/new-sub-charges
```
Comment (djorgensendk) [inline .github/workflows/ci.yml:27]: move pointer to look at main instead

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    with:
+      TERRAFORM_WORKING_DIR_PATH: './build/infrastructure/main'
+      TERRAFORM_VERSION: '1.0.10'
+
+  functionhost_ci:
+    needs: [ci_base, terraform_validate]
+    uses: Energinet-DataHub/.github/.github/workflows/dotnet-create-function-artifact.yml@1.13.0
+    with:
+      CSPROJ_FILE_PATH: 'source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.FunctionHost/GreenEnergyHub.Charges.FunctionHost.csproj'
+      DOTNET_VERSION: '5.0'
```
Comment (djorgensendk) [inline .github/workflows/ci.yml:50]: version 5.0.202 instead?
Perhaps introduce variabled instead

Comment (Renetnielsen) [reply .github/workflows/ci.yml]: Variables can't be used in the reusable workflows at the moment. Its a tradeoff of having centralized code vs having local variables.

I hope they will fix in the future

**Pull request merged**